cpp/merkletree/merkle_tree_test.cc

```diff
@@ -577,6 +577,12 @@ TEST_F(MerkleTreeTest, AddLeafHash) {
   EXPECT_EQ(kHashValue, tree.LeafHash(index));
 }
 
+TEST_F(CompactMerkleTreeTest, TestCloneEmptyTreeProducesWorkingTree) {
+  MerkleTree tree(new Sha256Hasher);
+  CompactMerkleTree compact(tree, new Sha256Hasher);
+  LOG(INFO) << compact.CurrentRoot();
```

Comment:
It could be, but what I'm checking here is that the whole damn thing doesn't just `abort()` anymore.